sos/report/plugins/openstack_instack.py

```diff
@@ -112,6 +100,21 @@ def setup(self):
             self.add_cmd_output("openstack object save "
                                 "tripleo-ui-logs tripleo-ui.logs --file -")
 
+    def capture_undercloud_yamls(self):
+        """ capture yaml files to define overrides """
+        uc_config = configparser.ConfigParser()
+        try:
+            uc_config.read(UNDERCLOUD_CONF_PATH)
+            override_opts = ['hieradata_override', 'net_config_override']
+            for opt in override_opts:
+                path = uc_config.get('DEFAULT', opt)
```

Comment:
These two "override" options are available under `DEFAULT` section as https://access.redhat.com/documentation/en-us/red_hat_openstack_platform/10/html/director_installation_and_usage/chap-installing_the_undercloud

So I pass the section argument to `DEFAULT`.